joomla/joomla-cms
# #11051 added tooltips

Pull Request for Issue #11051   .
#### Summary of Changes

Added tooltips for Article, Categories, Tags
#### Testing Instructions

» Apply PR
» Go to Article edit page > Publishing tab
» Hover on Created Date, Created By, Modified Date, Modified By Fields , you can see tooltips over there.
» Follow same steps for Categories and Tags.
